static/src/js/util/autocompleteFacetsMap.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * Maps the CMR Autocomplete types to CMR Facet types
+ */
+export const autocompleteFacetsMap = {
+  dataCenter: 'data_center_h',
+  instrument: 'instrument_h',
+  granuleDataFormat: 'granule_data_format_h',
+  platform: 'platform_h',
+  processingLevelId: 'processing_level_id_h',
+  project: 'project_h',
+  'science-keywords': 'science_keywords_h'
```

Comment:
Mentioned above, can we make this `scienceKeywords`? If it requires a change from CMR please write an additional ticket for us and link it to [CMR-6287](https://bugs.earthdata.nasa.gov/browse/CMR-6287).